doc/basics.md

```diff
@@ -44,6 +48,31 @@ Note:
 - Visual Studio users should compile with the `_CRT_SECURE_NO_WARNINGS` flag to avoid warnings with respect to our use of standard C functions such as `fopen`.
 
 
+
+
+Using simdjson as a CMake dependency
+------------------
+
+
```

Comment:
Can you remove these two newlines? And the two extras above the title? (Alternatively, we can keep all the distances the same.)

I'm so so sorry for this comment. It'll bug me forever though.

Reply:
Done.